infrastructure/cdn-in-a-box/edge/Dockerfile

```diff
@@ -23,7 +23,9 @@
 
 FROM centos:7
 
+ARG TO_API_FUNCTIONS=to-access.sh
```

Comment:
the api functions aren't something the user should be messing with since the rest of the initialization depends on it.  No need for that to be user-definable.